cli/src/cli/command/pipeline/service.py

```
@@ -0,0 +1,74 @@
+
+    await registry.initialize()
+
+    plugin_validation(pipeline_config, registry)
+
+    display_pipeline_info(pipeline_config)
+
+    client = await get_temporal_client(temporal_host, namespace)
+
+    actual_run_id = (
+        run_id
+        or f"cli_{datetime.now(tz=UTC).strftime('%Y%m%d_%H%M%S')}_{uuid4().hex[:8]}"  # noqa: E501
```

Ruff (0.12.2)

Unused `noqa` directive (non-enabled: `E501`)

Remove unused `noqa` directive

(RUF100)